lib/collection/src/operations/universal_query/shard_query.rs

```diff
@@ -56,6 +58,38 @@ impl ScoringQuery {
             _ => None,
         }
     }
+
+    /// Return the order of results, depending on the type of query
+    pub fn order(
```

Comment:
is this logic duplicated?

Reply:
It was! I will remove the one in CollectionConfig